tests/shouldwork/Issues/T2046BType.hs

```diff
@@ -0,0 +1,10 @@
+module T2046BType where
+
+import Clash.Prelude
+
+type T2046B =
+  ( (Index 1, Index 2)
+  , (Unsigned 1, Unsigned 2)
+  , (Signed 1, Signed 2)
+  , (BitVector 1, BitVector 2)
+  )
```

Comment:
Can you test `Bit` too, one of the functions changed for that

Reply:
It broke CI, good luck.

Reply:
```
            iverilog (make testBench):      FAIL (0.04s)
              Program /usr/bin/iverilog failed with error-code 2.
              
              Full invocation:
              
                /usr/bin/iverilog -I T2046BType.o -I T2046B.dyn_hi -I T2046B.hi -I T2046B.testBench -I T2046BType.dyn_o -I T2046BType.dyn_hi -I T2046B.topEntity -I T2046B.dyn_o -I T2046BType.hi -I T2046B.o -g2 -s testBench -o testBench.exe /tmp/clash-test-b57a279493374e3d/T2046B.testBench/testBench.v /tmp/clash-test-b57a279493374e3d/T2046B.topEntity/topEntity.v
                
              
              Stderr was:
              
                /tmp/clash-test-b57a279493374e3d/T2046B.topEntity/topEntity.v:98: syntax error
                /tmp/clash-test-b57a279493374e3d/T2046B.topEntity/topEntity.v:98: error: syntax error in continuous assignment
              
              Stdout was:
```